front/src/styles/common.css

```diff
@@ -4,35 +4,61 @@
   font-weight: 600;
   font-size: 14px;
   font-family: Open Sans;
+  font-style: normal;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
 h1 {
   font-size: 54px;
+  line-height: 92px;
 }
 
 h2 {
   font-size: 38px;
+  line-height: 63px;
 }
 
 h3 {
   font-size: 33px;
+  line-height: 53px;
 }
 
 h4 {
   font-size: 23px;
+  line-height: 34px;
 }
 
 h5 {
   font-size: 18px;
+  line-height: 27px;
 }
 
 h6 {
   font-size: 15px;
+  line-height: 21px;
 }
 
-.v-btn {
+body{
+  font-size: 14px;
```

Comment:
why we need to set font-size globally for password recovery?

Reply:
this is not for password recovery, but for everyone in General. in fact, you can delete it, but this is also taken from the layout